module/Frontpage/view/partial/poll.phtml

```diff
@@ -27,17 +27,31 @@ $lang = $this->plugin('translate')->getTranslator()->getLocale();
         <?php $text = $this->escapeHtml(($lang === 'en') ? $option->getEnglishText() : $option->getDutchText()); ?>
         <?php $voted = !is_null($userVote) && $option->getId() === $userVote->getPollOption()->getId(); ?>
 
-        <?php if ($showResults || !$canVote): ?>
-            <div>
-                <h5 class="clearfix<?= $voted ? ' text-primary' : '' ?>">
-                    <?= $text ?>
-                    <small class="pull-right" style="margin-top: 0.4em;"><?= $option->getVotesCount() === 1 ? $this->translate('1 vote') : sprintf($this->translate('%d votes'), $option->getVotesCount()) ?></small>
-                </h5>
+        <?php if ($showResults && $canVote): ?>
+            <div class="radio">
+                <label>
+                    <input type="radio" value="<?= $option->getId() ?>" name="option">
+                    <h5 class="clearfix<?= $voted ? ' text-primary' : '' ?>">
+                        <?= $text ?>
+                        <small class="pull-right" style="margin-top: 0.4em;"><?= $option->getVotesCount() === 1 ? $this->translate('1 vote') : sprintf($this->translate('%d votes'), $option->getVotesCount()) ?></small>
+                    </h5>
 
-                <div class="progress progress--poll">
-                    <div class="progress-bar" style="width: <?= $percentages[$option->getId()] ?>%;"></div>
-                </div>
+                    <div class="progress progress--poll">
+                        <div class="progress-bar" style="width: <?= $percentages[$option->getId()] ?>%;"></div>
+                    </div>
+                </label>
+            </div>
+        <?php elseif ($showResults || !$canVote): ?>
+        <div>
+            <h5 class="clearfix<?= $voted ? ' text-primary' : '' ?>">
+                <?= $text ?>
+                <small class="pull-right" style="margin-top: 0.4em;"><?= $option->getVotesCount() === 1 ? $this->translate('1 vote') : sprintf($this->translate('%d votes'), $option->getVotesCount()) ?></small>
+            </h5>
+
+            <div class="progress progress--poll">
+                <div class="progress-bar" style="width: <?= $percentages[$option->getId()] ?>%;"></div>
             </div>
+        </div>
```

Comment:
Missing indentation in comparison to all other code.

Reply:
Solved this